tests/wpt/README.md

```diff
@@ -199,13 +199,23 @@ next synced.
 Updating the upstream tests
 ===========================
 
-In order to update the tests from upstream use the same mach update
-commands. e.g. to update the web-platform-tests:
+In order to update the tests from upstream, use the same mach update
+commands. E.g. to update the web-platform-tests:
+
+    ./mach update-wpt --sync --patch
+
+Then, to update the test results:
 
-    ./mach update-wpt --sync
     ./mach test-wpt --log-raw=update.log
     ./mach update-wpt update.log
 
+You'll also need to do the same for the legacy layout engine:
+
+    ./mach test-wpt --legacy-layout --log-raw=legacy.log
+    ./mach update-wpt --legacy-layout legacy.log
+
+Finally, commit the results from `tests/wpt/meta/` and `tests/wpt/meta-legacy-layout/`.
+
 This should create two commits in your servo repository with the
 updated tests and updated metadata.
 
```

Comment:
I think this section can be removed entirely. This is about syncing the tests in the Servo repository with the ones from upstream, but this is now done by a CI job. 

At the same time the `--legacy-layout` option should be added to this section about updating test expectations: https://github.com/servo/servo/blob/1e315496f3db564ed440c9e28f61779768309013/tests/wpt/README.md#updating-test-expectations

Reply:
Makes sense! Updated: 8d892502cb688f64e922037194f58e99685a2d4e